nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.api.js

```diff
@@ -82,7 +82,7 @@ NfRegistryApi.prototype = {
      */
     getDroplet: function (bucketId, dropletType, dropletId) {
         var self = this;
-        var url = '/nifi-registry-api/buckets/' + bucketId + '/' + dropletType + '/' + dropletId;
+        var url = '../nifi-registry-api/buckets/' + bucketId + '/' + dropletType + '/' + dropletId;
```

Comment:
Looks like you missed one on line 58 (`getDropletSnapshotMetadata` still points to `/nifi-registry-api/`). 
Github doesn't let me comment on lines that were not edited so the comment is here.